nexus/api/graphql/query_run_resume_status.graphql

```diff
@@ -0,0 +1,23 @@
+query RunResumeStatus($project: String, $entity: String, $name: String!) {
+    model(name: $project, entityName: $entity) {
```

Comment:
I think we made aliases for `model` => `project` and `bucket` => `run` in all our resolvers.  I guess the annoying thing is our existing mock infra, but now's as a good as time as any to undo this 5 year old naming snafu 🙃